client/collector/src/main/java/org/hawkular/apm/client/collector/internal/ProcessorManager.java

```diff
@@ -440,7 +440,7 @@ public void process(Trace trace, Node node, Direction direction,
                 // Associate any issues created during initialisation with
                 // the node
                 if (issues != null) {
-                    node.getIssues().addAll(issues);
+                    node.getProperties().add(new Property(Constants.PROP_FAULT, issues));
```

Comment:
These should also be removed - not reported as a property. For now just add local logging.